docs/regressions/regressions-dl19-passage.splade-v3.cached.md

```diff
@@ -142,3 +142,4 @@ The experimental results reported here are directly comparable to the results re
 To add to this reproduction log, modify [this template](../../src/main/resources/docgen/templates/dl19-passage.splade-v3.cached.template) and run `bin/build.sh` to rebuild the documentation.
 
 + Results reproduced by [@xxx](https://github.com/xxx) on yyyy-mm-dd (commit [`xxxxxxx`](https://github.com/castorini/anserini/commit/xxxxxxx))
```

Comment:
Please follow the instructions above and modify the template? ^^^

Also, remove the `xxx` placeholder.